apps/condo/domains/ticket/components/IncidentForm/CreateIncidentForm.tsx

```diff
@@ -0,0 +1,47 @@
+import get from 'lodash/get'
+import React, { ComponentProps, useCallback, useMemo } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { useOrganization } from '@open-condo/next/organization'
+import { Button } from '@open-condo/ui'
+
+import ActionBar from '@condo/domains/common/components/ActionBar'
+import { Incident } from '@condo/domains/ticket/utils/clientSchema'
+
+import { BaseIncidentForm, BaseIncidentFormProps } from './BaseIncidentForm'
+
+
+export const CreateIncidentActionBar: React.FC<ComponentProps<BaseIncidentFormProps['ActionBar']>> = (props) => {
+    const intl = useIntl()
+    const SaveLabel = intl.formatMessage({ id: 'incident.form.save.label' })
+
+    const { handleSave, isLoading } = props
+
+    return (
+        <ActionBar>
+            <Button
+                type='primary'
+                children={SaveLabel}
+                onClick={handleSave}
+                disabled={isLoading}
+            />
+        </ActionBar>
+    )
+}
+
+
+export const CreateIncidentForm: React.FC = () => {
```

Comment:
That is not a FC component, because nothing happens if you pass children property to it)

Reply:
But what is it? :)

Reply:
This function returns React.ReactElement and takes nothing to it's arguments

Reply:
It seems to be normal practice

Reply:
By the way, `React 18` there is no longer a property `children` in  type `FC`)